processing/src/main/java/org/apache/druid/query/groupby/GroupByQueryConfig.java

```diff
@@ -75,11 +82,11 @@
   @JsonProperty
   // Size of on-heap string dictionary for merging, per-processing-thread; when exceeded, partial results will be
   // emitted to the merge buffer early.
-  private long maxSelectorDictionarySize = 100_000_000L;
+  private long maxSelectorDictionarySize = AUTOMATIC;
 
   @JsonProperty
   // Size of on-heap string dictionary for merging, per-query; when exceeded, partial results will be spilled to disk
-  private long maxMergingDictionarySize = 100_000_000L;
+  private long maxMergingDictionarySize = AUTOMATIC;
```

Comment:
I think we can change the type of `maxSelectorDictionarySize` and `maxMergingDictionarySize` from `long` to `HumanReadableBytes` to support human-readable format configuration.

Reply:
Good idea. I changed it.